Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM elifesciences/php_cli
+
+USER elife
+RUN mkdir -p /srv/proofreader
```

Comment:
uhm this folder should really be `/srv/proofreader-php` right?

Reply:
💯